mediathread/templates/base_new.html

```diff
@@ -80,28 +81,23 @@
 
         <div id="dialog-confirm" style="display: none"></div>
 
-                {% block masthead %}
-                    {% include "navbar.html" %}
-
-                    {% block courseinformation %}
-                        {% if not read_only_view %}
-                            <div id="course_information">
-                                <h1 id="course_title">
-                                    {% block coursetitle %}
-                                        <a href="{% url 'course_detail' course.pk %}"
-                                           title="{{course.title}}"
-                                           id="course_title_link">{{course.title}}</a>
-                                    {% endblock %}
-                                </h1>
-                                <div style="clear: right;"></div>
-                            </div>
-                        {% endif %}
-                    {% endblock %}
-                {% endblock %}
+        {% block masthead %}
+            {% include "navbar.html" %}
+        {% endblock %}
 
         {% block container %}
-            <div class="container">
-
+            <div class="container-fluid">
+                {% block courseinformation %}
+                    {% if not read_only_view %}
+                        <h6 class="course-title">
+                            {% block coursetitle %}
+                                <a href="{% url 'course_detail' course.pk %}"
```

Comment:
The wireframes don't show this as linkable...let's address this with Marc at our meeting later today.